### PR TITLE
LXQt: some rebuilds against qt5-5.8.0

### PR DESCRIPTION
**Pull request by gangsterakato**
This pull request fixes some graphical problems on LXQt since the update of Qt5. These packages (but `pavucontrol-qt`, which seems to make problems this time) must be rebuild against each new Qt5 update.

### CODE REVIEW

**Comment by ebfe** [comment]
One commit per template please.

`xrevbump` from the `xtools` package makes this easy. In this case:
 `xrevbump "rebuild against qt5-5.8.0" liblxqt libqtxdg lxqt-panel lxqt-qtplugin`

File: srcpkgs/liblxqt/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'liblxqt'
 pkgname=liblxqt
 version=0.11.1
-revision=1
+revision=3
```
Comment (Gottox) [inline srcpkgs/liblxqt/template]: 2

File: srcpkgs/libqtxdg/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libqtxdg'
 pkgname=libqtxdg
 version=2.0.0
-revision=3
+revision=5
```
Comment (Gottox) [inline srcpkgs/libqtxdg/template]: 4

File: srcpkgs/lxqt-panel/template [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'lxqt-panel'
 pkgname=lxqt-panel
 version=0.11.1
-revision=2
+revision=4
```
Comment (Gottox) [inline srcpkgs/lxqt-panel/template]: etc... 😄

**Comment by gangsterakato** [comment]
I will try to clean this mess.

**Comment by Gottox** [comment]
Don't close the PR, rebase your changes instead.

**Comment by gangsterakato** [comment]
I don't close it, but I didn't manage to squash and revert so I will reopen it with proper xrevbumps.

**Comment by pullmoll** [comment]
@gangsterakato never use `git pull` but always `git pull --rebase` to avoid the `Merge branch 'master' into <pkgname>`'.
You can remove the wrong merge using `git rebase -i HEAD~1` and `d`rop the offending commint.

**Comment by pullmoll** [comment]
See https://github.com/voidlinux/void-packages/commit/b4d1c053b7460b907c899902c87b919fd12da7ce and following

**Pull request closed without merging**